scripts/microbe_stage/setup.as

```diff
@@ -132,7 +132,16 @@ void applyEngulfMode(CellStageWorld@ world, ObjectID entity){
 void onReturnFromEditor(CellStageWorld@ world){
     LOG_INFO("TODO: apply the changes and spawn a copy of the player species from "
         "before the change");
-
+	//Increase the population by 30 and increase the generation
+	auto localPlayer = MicrobeOperations::getSpeciesComponent(world, "Default");
+	localPlayer.population += 30;
+	localPlayer.generation ++;
```

Comment:
I prefer the prefix notation `++localPlayer.generation` unless there is a reason to use the postfix. But I can let that slide.
But the space between `++` and the variable needs to be removed for consistency.